cmd/influxd/launcher/query_test.go

```diff
@@ -1047,6 +1068,27 @@ from(bucket: v.bucket)
 ,,0,1970-01-01T00:00:05Z,10,f,m0,k0
 ,,0,1970-01-01T00:00:10Z,22,f,m0,k0
 ,,0,1970-01-01T00:00:15Z,35,f,m0,k0
+`,
+		},
+		{
+			name: "sum with nulls",
+			query: `
+from(bucket: v.bucket)
+	|> range(start: 1970-01-01T00:00:00Z, stop: 1970-01-01T00:00:25Z)
+	|> aggregateWindow(every: 5s, fn: sum)
+	|> drop(columns: ["_start", "_stop"])
+`,
+			op: "readWindow(sum)",
+			want: `
+#datatype,string,long,dateTime:RFC3339,long,string,string,string
+#group,false,false,false,false,true,true,true
+#default,_result,,,,,,
+,result,table,_time,_value,_field,_measurement,k
+,,0,1970-01-01T00:00:05Z,10,f,m0,k0
+,,0,1970-01-01T00:00:10Z,22,f,m0,k0
+,,0,1970-01-01T00:00:15Z,35,f,m0,k0
+,,0,1970-01-01T00:00:20Z,5,f,m0,k0
+,,0,1970-01-01T00:00:25Z,,f,m0,k0
```

Comment:
Is count the only aggregate that implicitly fills null values?

Reply:
I've only looked at sum and count but sum will use a null value while count will use zero.